examples/particle_rayleigh_taylor_mu10/particle_rayleigh_taylor_mu10.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <testproblem>
   <name>Particle-rayleigh-taylor-mu10</name>
-  <owner userid="cmath"/>
+  <owner userid="drhodrid"/>
   <tags>flml</tags>
-  <problem_definition length="medium" nprocs="1">
+  <problem_definition length="long" nprocs="1">
```

Comment:
If it's changing to "long" it should move out into the longtests repo...

Reply:
Note that it's an example, not a test - so should probably stay where it is?